tools/load-test/README.md

```diff
@@ -0,0 +1,56 @@
+# Load Test
+
+
+## Overview
+
+The purpose of the load test is to verify the execution of the Horizontal Pod Autoscaling of functions deployed with the help of Kubeless in **Kyma**. It runs a **golang** app which stress a function by making thousands of HTTP POST request in set period of time defined in an environment variable. At the end of the execution of the test a notification is send to a Slack channel with the final output of the load test.
+
+## Installation
+
+```bash
+echo "Installing helm chart..."
+helm install --set slackEndpoint="${SLACK_ENDPOINT}" \
+             --set slackClientToken="${SLACK_CLIENT_TOKEN}" \
+             --set slackClientChannelId="${SLACK_CLIENT_CHANNEL_ID}" \
+             --set loadTestExecutionTimeout="$LT_TIMEOUT" \
+             ${LT_FOLDER_CHART} \
+             --namespace=kyma-system \
+             --name=load-test
+```
+
+The configuration needed to execute the **load test** is as follows:
+
+ | Name | Value | Description |
+ |------|---------------|-------------|
+**slackEndpoint** |`-`| A webhook slack url.
+**slackClientToken** |`-`|  A token which will be part of the **slackEndpoint**.
+**slackClientChannelId** |`#channelId`| ID of the Slack channel.
+**loadTestExecutionTimeout** |`30`| Time to finish the test otherwise it will timeout its execution. All the related metrics to be sent to the Slack channel are collected after the timeout.
+
+### Environment Variables
+
+To run the **load test** either in a cluster or a local Minikube you will need to set the above parameters for the installation of the helm chart. However in a cluster it is advisable to have environment variables as it is shown above.
```

Comment:
```suggestion
To run the load test either on a cluster or on a local Minikube, you must set the Helm chart installation parameters. However, on a cluster it is advisable to have environment variables set to default values.
```

Reply:
This section does not fit here. Please move it above the table with the environment variables.